src/20240528_AaveV3Ethereum_OnboardUSDeAaveV3Ethereum/OnboardUSDeAaveV3Ethereum.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Onboard USDe Aave V3 Ethereum"
+author: "ACI"
+discussions: "https://governance.aave.com/t/arfc-onboard-usde-to-aave-v3-on-ethereum/17690"
+snapshot: "https://snapshot.org/#/aave.eth/proposal/0xc1b6d0d390a2dabf81206f592f740c69163dd028dcb0f50182d0ad3a50e744b0"
+---
+
+## Simple Summary
+
+This proposal seek approval for the addition of USDe to Aave V3 on Ethereum
+
+## Motivation
+
+Ethena’s synthetic dollar, USDe, provides a stable crypto-native solution for a cash and carry structured product. The staked version of USDe, sUSDe, earns yield from the protocol and has high potential for strong borrow demand.
+
+## Specification
+
+The table below illustrates the configured risk parameters for **USDe**
+
+| Parameter                          |                                      Value |
+| ---------------------------------- | -----------------------------------------: |
+| Isolation Mode                     |                                       true |
+| Borrowable                         |                                    ENABLED |
+| Collateral Enabled                 |                                       true |
+| Supply Cap (USDe)                  |                                 80,000,000 |
+| Borrow Cap (USDe)                  |                                 72,000,000 |
+| Debt Ceiling                       |                                      USD 0 |
```

Comment:
40M in[ forum post](https://governance.aave.com/t/arfc-onboard-usde-to-aave-v3-on-ethereum/17690)